backend/src/main/java/gov/cdc/usds/simplereport/api/model/filerow/PatientUploadRow.java

```diff
@@ -138,10 +139,13 @@ public List<FeedbackMessage> validateIndividualValues() {
     errors.addAll(validateYesNoAnswer(employedInHealthcare));
     errors.addAll(validateRole(role));
 
-    // address
-    errors.addAll(validateState(state));
-    errors.addAll(validateZipCode(zipCode));
-    errors.addAll(validateCountry(country));
+    if (Boolean.FALSE.equals(
+        isAddressUnknown(state.getValue(), zipCode.getValue(), street.getValue()))) {
```

Comment:
this means we don't validate if they have valid unknown values for ALL 3 of these fields even if they included other address sections like city, street 2, etc